src/luacov/defaults.lua

```diff
@@ -73,4 +73,14 @@ return {
   -- Default: false.
   includeuntestedfiles = false,
 
+  --- Provides a function to parse filenames.
+  -- This function is used to parse filenames in the debug hook.
+  -- It is passed the filename as an argument and should return the parsed filename.
+  -- The default parser removes leading `@` and replaces path separators with `/`.
```

Comment:
The description here doesn't really match the behavior in the code below:

* in the code below, the @-stripping parser still takes precedence over the user's custom nameparser whenever a name starts with @, meaning that it is not only a "default" behavior that can be overridden with nameparser, but instead nameparser is just a fallback
* the slash-conversion behavior happens after the namesparser as well.

I think a less surprising behavior would be:
* priority order to be that the nameparser takes precedence over the @-stripping (that is, match what's implied in the docs when it calls the @-stripping the "default")
* slash-conversion can remain after nameparser entries for simplicity (that is, match what is in the code), but that needs to be documented